docs/content.zh/docs/dev/table/sql/queries/window-agg.md

```diff
@@ -199,7 +199,8 @@ The following shows a cascading window aggregation where the first window aggreg
 ```sql
 -- tumbling 5 minutes for each supplier_id
 CREATE VIEW window1 AS
-SELECT window_start, window_end, window_time as rowtime, SUM(price) as partial_price
+-- under the Cascading Window Aggregation to avoid field ambiguity window_start and window_end need to be renamed
```

Comment:
```suggestion
-- Note: The window start and window end fields of inner Window TVF are optional in the select clause. However, if they appear in the clause, they need to be aliased to prevent name conflicting with the window start and window end of the outer Window TVF.
```
WDTY?

Reply:
Looks good to me, I have revised